### DISCUSSION TITLE
Welcome to metasave Discussions!

### DISCUSSION THREAD

**Discussion by thuongtruong1009** [thuongtruong1009/metasave#20] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Ophelie22** [comment]
Hello , :) I tried to use your project. I make a .env like this.
 But I see only the test "Hello World" with the localhost:3000.
I would like to see the project , but i don't know what to do .
Can you explain me what i have to do ?



 APP_PORT=3000
APP_URL=http://localhost:${APP_PORT}

NODE_ENV=development

ORIGIN='http://localhost:3000,http://localhost:3001'

# Database
DB_NAME=metasave
MONGO_URL=mongodb://localhost:27017/metasave
ATLAS_URL=
